core/class.go

```diff
@@ -56,7 +56,28 @@ type Cairo1Class struct {
 	Program         []*felt.Felt
 	ProgramHash     *felt.Felt
 	SemanticVersion string
-	Compiled        json.RawMessage
+	Compiled        CompiledClass
+}
+
+type CompiledClass struct {
+	Bytecode        []*felt.Felt        `json:"bytecode"`
+	PythonicHints   json.RawMessage     `json:"pythonic_hints"`
+	CompilerVersion string              `json:"compiler_version"`
+	Hints           json.RawMessage     `json:"hints"`
+	EntryPoints     CompiledEntryPoints `json:"entry_points_by_type"`
```

Comment:
We can put the three entry point types in the `CompiledClass` struct directly. Wdyt?

Reply:
I mean move the three entry point fields into the `CompiledClass` struct and remove the `CompiledEntryPoints` type entirely.

Reply:
I got it, I do agree, it is much simpler.